Code/Editor/ViewportManipulatorController.cpp

```diff
@@ -72,13 +74,23 @@ namespace SandboxEditor
                 const auto* position = event.m_inputChannel.GetCustomData<AzFramework::InputChannel::PositionData2D>();
                 AZ_Assert(position, "Expected PositionData2D but found nullptr");
 
+                if (m_captureStart)
+                {
+                    m_virtualPosition = position->m_normalizedPosition;
+                    m_captureStart = false;
+                }
+                else
+                {
+                    m_virtualPosition += position->m_normalizedPositionDelta;
+                }
```

Comment:
suggestion: Don't have to make this change, but I wonder if this little bit of logic could actually be moved to a private function, it just feels like it'd be nice to wrap this up... maybe have it take `virtualPosition` as an inout parameter? Might not be worth it but just wanted to mention it in case...